README.rst

```diff
@@ -1,7 +1,26 @@
+flit
+====
+
+.. figure:: logo.png
+   :alt: The Flit Python packaging tool
```

Comment:
No, because GitHub doesn't support this either. See this similar issue for `asciidoc`: https://github.com/github/markup/issues/984

Basically what happens is:

  1. GitHub renderer calls `rest2html` on the `README.rst` file
  2. The HTML output now contains:
```html
<div class="figure align-right">
<img alt="The Flit Python packaging tool" src="logo.png" />
</div>
 ```
   3. The HTML is nuked through GitHub's HTML Pipeline [sanitization filter](https://github.com/jch/html-pipeline/blob/master/lib/html/pipeline/sanitization_filter.rb), which strips off all `style` definitions and `class` attributes, resulting in this HTML markup actually being renderer:
```html
<div>
<img alt="The Flit Python packaging tool" src="logo.png" />
</div>
 ```